reference/6/Microsoft.PowerShell.Core/New-PSSession.md

```diff
@@ -92,6 +92,14 @@ The benefit of an SSH based PowerShell remote session is that it can work across
 For SSH based sessions you use the **HostName** or **SSHConnection** parameter set to specify the remote computer and relevant connection information.
 For more information about how to set up PowerShell SSH remoting, see [PowerShell Remoting Over SSH](/powershell/scripting/core-powershell/ssh-remoting-in-powershell-core).
 
+> [!NOTE]
+> When using WSMan remoting from a Linux or macOS client with a HTTPS endpoint where
```

Comment:
not sure how applicable this information is, but just FYI:
I had to use these flags from Windows client as well (during Redfish work).

Reply:
Ignore this, in my case it was not New-pssession.